api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -100,12 +108,34 @@ private MandrillMessage buildWelcomeMessage(String contactEmail, String password
     RecipientAddress toAddress = new RecipientAddress();
     toAddress.setEmail(contactEmail);
     msg.setTo(Collections.singletonList(toAddress));
-    String msgBody = "Your new account is: " + user.getPrimaryEmail() +
-      "\nThe password for your new account is: " + password;
-    msg.setHtml(msgBody);
-    msg.setSubject("Your new All of Us Account");
-    msg.setFromEmail(workbenchConfigProvider.get().mandrill.fromEmail);
-    return msg;
+    try {
+      String msgHtml = buildEmailHtml(password, user);
+      msg.setHtml(msgHtml);
+      msg.setSubject("Your new All of Us Account");
+      msg.setFromEmail(workbenchConfigProvider.get().mandrill.fromEmail);
+      return msg;
+    } catch (IOException e) {
+      throw new MessagingException("Error reading in email");
+    }
+  }
+
+  private String buildEmailHtml(String password, User user) throws IOException {
+    CloudStorageService cloudStorageService = cloudStorageServiceProvider.get();
+    StringBuilder contentBuilder = new StringBuilder();
+    URL emailContent = Resources.getResource(WELCOME_RESOURCE);
+    Resources
+        .readLines(emailContent, StandardCharsets.UTF_8)
+        .forEach(s -> contentBuilder.append(s).append("\n"));
+    String string = contentBuilder.toString();
+    Map<String, String> replaceMap = new HashMap<>();
```

Comment:
opt_nit: I'd probably use an ImmutableMap.Builder here so I could chain these puts (and then inline that in the StrSubstitutor ctor call).